src/main/java/io/github/perrymant/moneymaker/MoneyFormatter.java

```diff
@@ -0,0 +1,15 @@
+package io.github.perrymant.moneymaker;
+
+import java.text.NumberFormat;
+import java.util.Locale;
+
+class MoneyFormatter {
+    private static final Locale UK = Locale.UK;
+
+    static String formatPounds(int amount) {
+        final double value = (double) amount / 100;
+        NumberFormat currencyInstance = NumberFormat.getCurrencyInstance(UK);
+        return currencyInstance.format(value);
```

Comment:
Can inline this value. Having `value` as a variable name is a bit of a code smell. Can almost always (exception: key-value pairs in maps) use a better variable name, or avoid this variable entirely.